web/models/setting/setting.ex

```diff
@@ -0,0 +1,37 @@
+defmodule CoursePlanner.Setting do
+  @moduledoc """
+  This module holds the model for the setting table
+  """
+  use CoursePlanner.Web, :model
+
+  schema "settings" do
+    field :notification_frequency, :integer
```

Comment:
this is rather the notification period, frequency is more like 3 notifications/day